### DISCUSSION TITLE
Feature Request: specify card type in markdown file

### DISCUSSION THREAD

**Discussion by Crafty5546** [jasonwilliams/anki#124] [Q&A]
Hi, I really like this extension. 

I'm wondering if there is a way to change the card type within the markdown file itself? There are a few cards that I want to be reverse cards and some I want as normal card type. Or maybe include an extension command to send to Anki as Reverse Type. Anki: Send To Deck (Reverse Card Type)?

**Comment by jasonwilliams** [comment]
Hi @Crafty5546 that sounds reasonable, I've converted this into an [issue](https://github.com/jasonwilliams/anki/issues/125)
